encoder.py

```diff
@@ -145,11 +120,13 @@ def _compute_embeddings(self, docs_batch_generator: Iterable) -> None:
                 else:
                     images = np.stack(blob_batch)
                 tensor = torch.from_numpy(images).to(self.device)
-                features = self._get_features(tensor).detach()
-                features = self._get_pooling(features.cpu().numpy())
+                features = self._get_features(tensor)
```

Comment:
let's merge this into a function `_get_embedding`

Reply:
Done in https://github.com/jina-ai/executor-image-torch-encoder/pull/7/commits/9631c630ccfca61e2cbe9f4c5efb5ce1178de0c8